react-ai-chatbot/src/components/Controls/Controls.module.css

```
@@ -19,6 +19,7 @@
     outline: none;
     resize: none;
     background: #ffffff;
+    background: light-dark(#ffffff, #212121);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)